src/main/scala/beam/agentsim/agents/ridehail/allocation/EVFleetAllocationManager.scala

```diff
@@ -62,16 +60,18 @@ class EVFleetAllocationManager(val rideHailManager: RideHailManager)
         // from future consideration if it fails this check
         val routingResponses = routeResponsesByDriver.get(agentLocation.vehicleId)
 
+        val routedDisanceInM = routingResponses
+          .getOrElse(List[RoutingResponse]())
```

Comment:
Why not just make this a fold also?

Reply:
cause I'm not a scala ninja :)

Reply:
Fair point! I should word my comments better :)